Fix spelling mistakes and re-word sentences for clarity

0x41c opened the pull request.

0x41c committed:
Just a couple punctuation thingys

0x41c committed:
Some more refactoring

0x41c committed:
More re-sentancing and small touches of formatting

rickmark commented on EFI.md:
```diff
@@ -16,13 +16,13 @@ T2 systems boot EFI like their predicessors.  Unlike previous versions they use
-However, you can chainload to another bootloader if you want to, it's not prevented when Secure Boot is set to permissive.
-That permissive option doesn't technically turn off Secure Boot. It allows you to enroll your own hashes to the Secure Boot policy on the system w/ user authentication instead.
+Apple Silicon chips don't run EFI but run iBoot instead (Though it does emulate parts of the EFI system via IOKit for back compatibility).
+Even though it's iboot, you can still chainload to another bootloader if you want to; It's not prevented when Secure Boot is set to permissive.
```
iBoot casing

rickmark commented on EFI.md:
```diff
@@ -16,13 +16,13 @@ T2 systems boot EFI like their predicessors.  Unlike previous versions they use
-That permissive option doesn't technically turn off Secure Boot. It allows you to enroll your own hashes to the Secure Boot policy on the system w/ user authentication instead.
+Apple Silicon chips don't run EFI but run iBoot instead (Though it does emulate parts of the EFI system via IOKit for back compatibility).
+Even though it's iboot, you can still chainload to another bootloader if you want to; It's not prevented when Secure Boot is set to permissive.
+That permissive option doesn't technically turn off Secure Boot, but allows you to enroll your own hashes to the Secure Boot policy on the system w/ user authentication instead.
```
Confusing "permissive" with "fuOS".  Permissive puts XNU in a legacy intel like security state.  fuOS or `coih` is the chain loading mechanism

rickmark replied:
`coih` / `fuOS` - chainloading
`smb0` - `smb3` - Secure Multi Boot - iOS like immutable kernel, these are switched on for "permissive" to have MDM and kext loading
`sip0` - `sip3` - classical SIP settings

all verified by iBoot before execution is passed to XNU

rickmark commented on README.md:
```diff
@@ -7,7 +7,7 @@ Join us on Discord: https://discord.gg/hackdifferent
 
 ## WARNING
-This repo accepts (gleefully) PRs that enhance and enable development, but under no circumstances create a PR based on `AppleInternal`, or any other copyrighted workds protected by the [DMCA](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act).  If you need help determining this tag the PR and join the Discord server and ask a `#Legit` or higher role for help.
+This repo accepts (gleefully) PRs that enhance and enable development, but under no circumstances create a PR based on `AppleInternal`, or any other copyrighted words protected by the [DMCA](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act).  If you need help determining this, tag the PR, join the Discord server, and ask a `#Legit` or higher role for help.
```
tag the PR with "license help"

0x41c committed:
Fix my own issues lmao

0x41c committed:
Multiple changes enforced by markdownlint and Grammarly

Fixed issues outlined in the review to the PR and corrected various spelling mistakes in other places to the best of my ability.
